docker-build.sh

```diff
@@ -13,4 +23,4 @@ popd
 docker build --rm -f "Dockerfile-stage2" -t ug-chromium-builder-stage2:latest ./
 
 # start the browser build
-docker run -ti -v `pwd`/build:/repo/build ug-chromium-builder-stage2:latest bash -c "./build.sh && ./package.sh"
+docker run $NCPUS -ti -v `pwd`/build:/repo/build ug-chromium-builder-stage2:latest bash -c "$PRECMD && ./build.sh && ./package.sh && $POSTCMD"
```

Comment:
Instead of adding PRECMD and POSTCMD here, why not just have a single `BUILDCMD` that by default consists of `./build.sh && package.sh`?